### PR TITLE
Heads of Staff now spawn with handguns

### PR DESCRIPTION
**Pull request by ValoTheValo**
## About The Pull Request
This pull request makes heads of staff spawn with a handgun with 2 magazines of rubber ammo, intended to be merged with #7130 
MEO and MBO Get a Anne with 2 magazines of rubber .35 ammo
NT Preacher gets a Svallin with 2 NT Cells, but now spawns with them in inventory (Same as Before)
TE Gets a Makarov with 2 magazines of rubber .35 ammo
Merchant gets a Olivaw with 2 Magazines of rubber .35 ammo
Captain and FO Spawn with Avas in their inventory instead of locker with 1 magazine of rubber (3 more are in lockers)

## Why It's Good For The Game
Gives heads of staff abit more authority and personal protection, will also help in cases of early round fights and not being spawnkilled

## Changelog
:cl:
add: All Heads of staff now spawn with handguns and 2 rubber magazines
del: FO and Captains pistols no longer spawn in their locker
/:cl:

### CODE REVIEW

**Comment by Humonitarian** [comment]
I'd rather the guns be kept in their closets, personally.
Paramedic Annes should be moved in their closets as well.

File: code/datums/outfits/jobs/church.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	suit = /obj/item/clothing/suit/storage/neotheology_coat
 	shoes = /obj/item/clothing/shoes/reinforced
 	gloves = /obj/item/clothing/gloves/thick
-	backpack_contents = list(/obj/item/book/ritual/cruciform/priest = 1, /obj/item/clothing/accessory/cross = 1)
+	backpack_contents = list(/obj/item/book/ritual/cruciform/priest = 1, /obj/item/clothing/accessory/cross = 1,/obj/item/gun/energy/nt_svalinn = 1,/obj/item/cell/small/neotheology/plasma = 2)
```
Comment (Firefox13) [inline code/datums/outfits/jobs/church.dm:21]:
```suggestion
	backpack_contents = list(/obj/item/book/ritual/cruciform/priest = 1, /obj/item/clothing/accessory/cross = 1, /obj/item/gun/energy/nt_svalinn = 1, /obj/item/cell/small/neotheology/plasma = 2)
```

File: code/datums/outfits/jobs/engineering.dm [hunk 2]
```diff
@@ -20,6 +20,7 @@
 	l_ear = /obj/item/device/radio/headset/heads/ce
 	id_type = /obj/item/card/id/ce
 	pda_type = /obj/item/modular_computer/pda/heads/ce
+	backpack_contents = list(/obj/item/gun/projectile/selfload/makarov = 1,/obj/item/ammo_magazine/pistol/rubber = 2) //TE got the excel gun as a war trophy same as the hatton
```
Comment (Firefox13) [inline code/datums/outfits/jobs/engineering.dm:23]:
```suggestion
	backpack_contents = list(/obj/item/gun/projectile/selfload/makarov = 1, /obj/item/ammo_magazine/pistol/rubber = 2) //TE got the excel gun as a war trophy same as the hatton
```

File: code/datums/outfits/jobs/guild.dm [hunk 3]
```diff
@@ -12,6 +12,7 @@
 	id_type = /obj/item/card/id/car
 	pda_type = /obj/item/modular_computer/pda/cargo
 	l_ear = /obj/item/device/radio/headset/heads/merchant
+	backpack_contents = list(/obj/item/gun/projectile/olivaw = 1,/obj/item/ammo_magazine/pistol/rubber = 2)
```
Comment (Firefox13) [inline code/datums/outfits/jobs/guild.dm:15]:
```suggestion
	backpack_contents = list(/obj/item/gun/projectile/olivaw = 1, /obj/item/ammo_magazine/pistol/rubber = 2)
```

File: code/datums/outfits/jobs/science.dm [hunk 4]
```diff
@@ -19,7 +19,7 @@
 	l_hand = /obj/item/clipboard
 	id_type = /obj/item/card/id/rd
 	pda_type = /obj/item/modular_computer/pda/heads/rd
-	backpack_contents = list(/obj/item/oddity/secdocs = 1)
+	backpack_contents = list(/obj/item/oddity/secdocs = 1,/obj/item/gun/projectile/selfload/moebius = 1,/obj/item/ammo_magazine/pistol/rubber = 2)
```
Comment (Firefox13) [inline code/datums/outfits/jobs/science.dm:22]:
```suggestion
	backpack_contents = list(/obj/item/oddity/secdocs = 1, /obj/item/gun/projectile/selfload/moebius = 1, /obj/item/ammo_magazine/pistol/rubber = 2)
```

File: code/datums/outfits/jobs/medical.dm [hunk 5]
```diff
@@ -21,6 +21,7 @@
 	id_type = /obj/item/card/id/cmo
 	pda_type = /obj/item/modular_computer/pda/heads/cmo
 	belt = /obj/item/storage/belt/medical/
+	backpack_contents = list(/obj/item/gun/projectile/selfload/moebius = 1,/obj/item/ammo_magazine/pistol/rubber = 2)
```
Comment (Firefox13) [inline code/datums/outfits/jobs/medical.dm:24]:
```suggestion
	backpack_contents = list(/obj/item/gun/projectile/selfload/moebius = 1, /obj/item/ammo_magazine/pistol/rubber = 2)
```

**Comment by TheShown911** [comment]
They already spawn with the ability to grab, should be enough for them.

**Review by Firefox13 — APPROVED** [review]
Grab-fu is overrated

**Pull request merged**